pkg/agent/controller/egress/egress_controller.go

```diff
@@ -196,6 +205,18 @@ func NewEgressController(
 	if trafficShapingEnabled && !openflow.OVSMetersAreSupported() {
 		klog.Info("EgressTrafficShaping feature gate is enabled, but it is ignored because OVS meters are not supported.")
 	}
+
+	eventBroadcaster := record.NewBroadcaster()
+	eventBroadcaster.StartStructuredLogging(0)
+	runtime.Must(scheme.AddToScheme(scheme2.Scheme))
```

Comment:
This doesn't seem right. Egress has nothing to do with multicluster.

And scheme is initialize in `init` func, not at runtime.

Reply:
yes i have pushed the wrong import of scheme package here will correct that.